### DISCUSSION TITLE
蓝灯充值一直没到账

### DISCUSSION THREAD

**Discussion by liming123179** [getlantern/lantern-forum-cn#1069] [General]
蓝灯充值一直没到账，44668423@qq.com
![360截图20241203083943286](https://github.com/user-attachments/assets/4e2eafaa-0dc3-43f4-aac8-182787837bad)

**Comment by MaxMKMax** [comment]
@liming123179 您好. 好的, 我会帮您提交上去处理. 谢谢您的耐心等待.

**Comment by liming123179** [comment]
你好，您的邮件我已收到！
